data/card_catalogue.json

```diff
@@ -1068,27 +1030,27 @@
         {
           "title": "Hope",
           "upgrade": "Pray",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Change the value of a thrown d20 or the next d20 roll by 3",
+            "rules": "Change the value of thrown d20 or the next thrown d20 by 2",
             "flavour": "I can't see him, but I know he's there.",
             "image": "assets/core/card/illustration/Hope.png"
           }
         },
         {
           "title": "Pray",
           "upgrade": "Preach",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "For the next 3 d20 rolls this turn, you may force them to be rerolled",
+            "rules": "You may force a thrown d20 or the next thrown d20 to be rerolled.",
```

Comment:
This wording blows, but I didn't come up with a nicer way to word it that works in both the non-combat and combat situations